docs/releases/dev.rst

```diff
@@ -22,7 +22,7 @@ Major Changes
 
 - Updated translations.
 - Added django-transaction-hooks
-
+- Changed user delete behaviour
```

Comment:
This is pretty uninformative. Add more details for third party people to learn something useful.